actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1879,6 +1879,7 @@ def submit_default_value
           end
 
           defaults = []
+          defaults << :"helpers.submit.#{object.model_name.i18n_key}.#{key}" if object.respond_to?(:model_name)
```

Comment:
Instead of adding a new lookup can we add the `:"helpers.submit.#{object.model_name.i18n_key}.#{key}"` if it object responds to `model_name` and `defaults << :"helpers.submit.#{object_name}.#{key}"` if it doesn't?